config/values/finance.yml

```diff
@@ -1,4 +1,5 @@
 finance:
+  academicyear: 2025 to 2026 academic year
```

Comment:
I don't know whether all of these financial amounts are updated at the same time (although they might well be) - if we have 25/26 figures for all of them now, this is OK!